### PR TITLE
Added a TASK_STATUS  - unassigned to constants

### PR DESCRIPTION
**Pull request by ritikrana4**
Updated the TASK_STATUS constant with a new value "UNASSIGNED: unAssigned".

ISSUE: Update task constants #366

@swarajpure 
@whyDontI

### CODE REVIEW

File: controllers/members.js [hunk 1]
```diff
@@ -33,8 +33,8 @@ const getIdleMembers = async (req, res) => {
   try {
     const allMembers = await fetchMembers()
     const taskParticipants = await tasks.fetchActiveTaskMembers()
-    const idleMembers = allMembers?.filter(({ id }) => !taskParticipants.has(id))
-    const idleMemberUserNames = idleMembers?.map((member) => member.username)
+    const idleMembers = allMembers.filter(({ id }) => !taskParticipants.has(id))
+    const idleMemberUserNames = idleMembers.map((member) => member.username)
```
Comment (swarajpure) [inline controllers/members.js:37]: Why was this removed? We need those checks in place
It's not by mistake, it's called optional chaining: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

You maybe having older version of Node, that's why it didn't work in your machine, try updating node in your machine

File: constants/tasks.js [hunk 2]
```diff
@@ -8,7 +8,8 @@ const TASK_STATUS = {
   ACTIVE: 'active',
   ASSIGNED: 'assigned',
   BLOCKED: 'blocked',
-  COMPLETED: 'completed'
+  COMPLETED: 'completed',
+  UNASSIGNED: 'unAssigned'
```
Comment (swarajpure) [inline constants/tasks.js:12]: Cool!

**Review by swarajpure — CHANGES_REQUESTED** [review]
Please revert the change of removing optional chaining operators, we can then merge this PR 🙂

File: controllers/members.js (hunk 1 above)
Comment (ritikrana4) [reply controllers/members.js]: ok. Did not know about that. 👍

**Comment by ritikrana4** [comment]
Reverted the changes and has added optional chaining. 
@swarajpure Please review.

**Review by swarajpure — APPROVED** [review]
Thanks for the change!

You might want to join our Discord server where we (Real Dev Squad) have all the technical and non-technical discussions. You can get more tasks, increase your knowledge and upskill yourself 😉 

Join discord server here: https://discord.gg/a3hBt2PZ
